tests/e2e/tests/pilot/istio_rbac_v2_test.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 Istio Authors
```

Comment:
May be we should start add tests in the test/integration folder with the new framework instead. @lei-tang , what is the consensus and the plan to do so? (i.e do we have to do in both place or migrate all at once)

Reply:
Based on Nathan's email, it is recommended to add the tests to the tests/integration folder, which will replace the stuff in the tests/e2e folder. I have forwarded Nathan's email to Phillip.

Reply:
We can get this in and get ready to migrate the test to the new framework later.

@lei-tang could you also forward the email to me? Thanks.

Reply:
@yangminzhu the email has been forwarded to you.

Reply:
Thanks Lei

Reply:
Have we reached an agreement yet? 

@ozevren Care to comment?

Reply:
The choice is upto you, but I'd strongly recommend writing new tests using the new integration test framework. It allows you to write tests to run on bth Kubernetes, as well as in local execution mode. I am planning to enable the gates as required by the end of this week.

Reply:
@diemtvu Do you mind if we merge this for now?

Reply:
sgtm